### PR TITLE
RDoc-1519 - proofread

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown [hunk 1]
```diff
@@ -1,85 +1,85 @@
...
+   * Input the connection string to your MongoDB instance 
+   
+4. **Migrate GridFS** 
+   * Choosing this option will import GridFS attachments and save them as documents with attachments in the `@files` collection
+   
+5. **Database Name**
+   * Input the name of the source database
+   
+6. **Migrate all collections**
+   * You can either import all collections or choose collections individually
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown]: or choose collections individually => or select specific collections

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown [hunk 2]
```diff
@@ -1,85 +1,85 @@
...
+   * URL for CosmosDB database 
+   
+4. **Primary Key** 
+   * Primary key for CosmosDB
+   
+5. **Database Name**
+   * Input the name of the source database
+   
+6. **Migrate all collections**
+   * You can either import all collections or choose collections individually
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown]: or choose collections individually => or select specific collections

**Comment by morhilai** [comment]
Will fix

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/settings/import-from-other.markdown]: Done

**Pull request merged**